sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py

```diff
@@ -2,8 +2,10 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-from typing import cast, Union
-from xml.etree.ElementTree import ElementTree
+from datetime import datetime, timedelta
+from typing import cast
+from xml.etree.ElementTree import ElementTree, SubElement, QName
+import isodate
```

Comment:
have we used this package before? I can't recall.

should we add dependency requirement in setup.py or it's already installed by other libs we rely on.

Reply:
I chose isodate because msrest uses it. I added it into the dependency in SB setup.py in case that msrest remove it someday.